File: Java/Data Structures & Algorithms Module - Java/Week4/Stacks/Node.java
/*
Node.java
Billy Cussen
16/09/2020
*/

public class Node<T>{

    T element;
    Node<T> next;

    public Node(T element){
        this.element = element;
    }

    public Node(T element, Node<T> next){
        this.element = element;
        this.next = next;
    }

}

File: Python/Data Structures & Algorithms Module - Python/Week10/BST Attempt 2/Node.py
"""
Node.py
Billy Cussen
27/11/2020
"""

class Node:
    def __init__(self,element):
        self.element = element
        self.left = None
        self.right = None

    def __str__(self):
        leftNode = "NULL" if self.left == None else self.left.element
        rightNode = "NULL" if self.right == None else self.right.element
        return "Current: "+str(self.element)+", Left: "+str(leftNode)+", Right: "+str(rightNode)